app/dao/template_categories_dao.py

```diff
@@ -0,0 +1,91 @@
+import uuid
+
+from flask import current_app
+
+from app import db
+from app.dao.dao_utils import transactional
+from app.models import Template, TemplateCategory
+
+
+@transactional
+def dao_create_template_category(template_category: TemplateCategory):
```

Comment:
do we need validation here? If the fields that are required like name/ etc are empty - then we should bubble the error up? or we could do in the API

Reply:
This is covered by the [schema class](https://github.com/cds-snc/notification-api/pull/2193/files#diff-743c8db0af830ae358f828cadf2faa7caaa075029cdce9bd4650ed51ca80038f) when we [call `load` on the request json](https://github.com/cds-snc/notification-api/pull/2193/files#diff-0ad60abb8684d413a0a5c260f18655d434359f538388c5d8e0f5affc3db8624cR28) it will raise a `ValidationError` when the values are invalid.